content/en/docs/tasks/administer-cluster/verify-signed-images.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Verify Signed Container Images
+
+reviewers:
+  - TBD
+
+content_type: task
+min-kubernetes-server-version: v1.24
+---
+
+<!-- overview -->
+
+{{< feature-state state="alpha" for_k8s_version="v1.24" >}}
+## Pre-requisites
+- [cosign](https://docs.sigstore.dev/cosign/installation/)
+
+## Verifying image signatures
+For a complete list of images that are signed please refer to [releases](../../../../releases/download) page.
+
+Let's pick one image from this list and verify its signature using `cosign verify` command.
+
+```shell
+COSIGN_EXPERIMENTAL=1 cosign verify k8s.gcr.io/kube-apiserver-amd64:v1.24.0
+```
+### All control plane images
+
+To verify all signed control plane images, please run this command
```

Comment:
nit: A period is missing.

```suggestion
To verify all signed control plane images, please run this command.
```

Reply:
I think it would be a comma, not a period @jihoon-seo

Reply:
How about a colon? 😊